pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarFunctionE2ETest.java

```diff
@@ -217,6 +256,9 @@ void setup(Method method) throws Exception {
             if (connectorsDir.mkdir()) {
                 File file = new File(getClass().getClassLoader().getResource("pulsar-io-data-generator.nar").getFile());
                 Files.copy(file.toPath(), new File(connectorsDir.getAbsolutePath() + "/" + file.getName()).toPath());
+                
+                file = new File(getClass().getClassLoader().getResource("pulsar-io-batch-data-generator.nar").getFile());
```

Comment:
Since you are copying the connector into the "connectors" dir, please also add a test to test submitting the builtin version of the connector.